src/bundle/Resources/public/scss/fieldType/edit/_ezobjectrelationlist.scss

```diff
@@ -47,7 +47,7 @@
 
             .btn {
                 line-height: .85;
-                padding: .5rem .55rem;
+                padding: 0.5rem 0.3rem .5rem .6rem;
```

Comment:
CS issue: please follow one convention of setting values. I suggest to follow this convention: `0.5rem 0.3rem 0.5rem 0.6rem`. What's more there's our custom SASS function: `calculateRem()` which takes one param. It basically converts pixels to rems. For instance:
```scss
.some-class {
  height: calculateRem(12px);
}
```
Converts it to:
```css
.some-class {
  height: 0.75rem;
}
```
Because the base font size is `16px`.
Can you reuse that function in this line?

Reply:
Added fixup based on comment.

![ez platform 21](https://user-images.githubusercontent.com/9256718/43782652-cfd809b0-9a2d-11e8-8a4a-e5faac52517f.png)